tests/batching/test_hooks.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from mlserver.model import MLModel
+from mlserver.types import InferenceRequest, InferenceResponse
+from mlserver.batching.hooks import load_batching
+
+
+async def test_batching_predict(
+    sum_model: MLModel, inference_request: InferenceRequest
+):
+    await load_batching(sum_model)
+    response = await sum_model.predict(inference_request)
+
+    assert response is not None
+    assert isinstance(response, InferenceResponse)
+    assert len(response.outputs) == 1
+
+
+@pytest.mark.parametrize(
+    "max_batch_size, max_batch_time",
+    [
+        (0, 10),
+        (-1, 10),
+        (1, 10),
+        (10, 0),
+        (10, -1),
+        (0, 0),
+    ],
+)
+async def test_load_batching_disabled(
```

Comment:
is batching disabled because either max_batch_size or max_batch_time is not?

how are you making sure that actually batching is disabled and not exercised?

Reply:
Depending on the values of `max_batch_size` and `max_batch_time`, we disable adaptive batching. On all the cases covered by that `pytest.mark.parametrize` above, the expectation is that adaptive batching should get disabled.